pandas/api/__init__.py

```diff
@@ -1,2 +1,2 @@
-""" public toolkit API """
+"""public toolkit API"""
```

Comment:
So I would say generally just completely revert any changes to module-level docstrings. These are most likely harmless but at the same time have zero upside right now to changing, so would be better to revert entirely and have a smaller diff

Reply:
Reverted all the module-level docstring changes (Y)

Reply:
Reverted all changes outside of pandas.core. Trust this sorts it out.